core/arch/arm/include/kernel/thread.h

```diff
@@ -628,6 +628,25 @@ uint32_t thread_rpc_cmd(uint32_t cmd, size_t num_params,
 unsigned long thread_smc(unsigned long func_id, unsigned long a1,
 			 unsigned long a2, unsigned long a3);
 
+/**
+ * Allocates data for payload buffers.
+ * Buffer is exported using dma-buf API
+ *
+ * @size:	size in bytes of payload buffer
+ * @cookie:	returned cookie used when freeing the buffer
+ *
+ * @returns	mobj that describes allocated buffer or NULL on error
+ */
+struct mobj *thread_rpc_alloc_dmabuf_payload(size_t size, uint64_t *cookie);
```

Comment:
@jenswi-linaro I guess here the same problem (I need to avoid "dma-buf"), as we discussed in https://github.com/linaro-swg/linux/pull/58

Reply:
Yes, I think `s/dmabuf/global/` should do it.